articles/as_long_as.md

```diff
@@ -0,0 +1,29 @@
+---
+authorName: Sabrina Lange
+authorGithubUsername: sweinzie
+issue: 96
+title: “As long as you put your Feet Under my Table” - If the Project Manager Leads Differently than I Would 
+---
+# {{page.title}}
+
+“As long as you put your Feet Under my Table” - If the Project Manager Leads Differently than I Would 
```

Comment:
1. "solange du die füße unter meinen tisch stellst" würde ich übersetzen mit "As long as you live under my roof, you'll do as I say". 
2. Macht der Satz hier Sinn?